tests/test_itrf_converter.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import pandas as pd
+
+from iceflow.itrf.converter import _datetime_to_decimal_year, transform_itrf
+
+
+def test_transform_itrf():
+    synth_df = pd.DataFrame(
+        {
+            # Note: duplicate data here because otherwise a deprecation warning
+            # error about single-value series is raised from pandas' internals,
+            # and pytest complains.
+            "latitude": [70, 70],
+            "longitude": [-50, -50],
+            "elevation": [1, 1],
+            "ITRF": ["ITRF93", "ITRF93"],
+        },
+    )
+
+    constant_datetime = pd.to_datetime("1993-07-02 12:00:00")
+    synth_df.index = pd.Index([constant_datetime] * 2, name="utc_datetime")
+
+    result = transform_itrf(
+        data=synth_df,
+        target_itrf="ITRF2014",
+    )
+
+    # Before and after values were extracted from the synthetic dataset in the
+    # ITRF corrections notebook put together by Kevin. See:
+    # https://github.com/nsidc/NSIDC-Data-Tutorials/blob/a35203f18841456d258fac3a11dc04f44f839d9d/notebooks/iceflow/corrections.ipynb
```

Comment:
I found the datetime conversion bug because the notebook hard-codes the decimal year as 1993.5 - it doesn't calculate it from a datetime like we normally do for real data.